desc/objectives/_coils.py

```diff
@@ -1288,6 +1292,8 @@ def __init__(
         vacuum=False,
         name="Quadratic flux",
         jac_chunk_size=None,
+        *,
+        chunk_size=1,
```

Comment:
For coils this is likely way too small and will slow things down on GPU.

In general, for coils or other 1d sources I don't expect that we'd ever really need chunking, except in maybe the most extreme cases.
Even for 2d sources, I'm surprised it's necessary? At what point do you run out of memory?

Reply:
chunk_size=1 is basically the same as master. Some optimization regression tests were memory hungry with full chunking, so I reverted the objectives to chunk size 1

Reply:
and by the way, i run things on cpu on my laptop, and large chunks still improve performance, so it's not just gpu

Reply:
I made the request. If things break because of this, please revert/fix [d028bdd](https://github.com/PlasmaControl/DESC/pull/1568/commits/d028bdd6c3a15cfc414f26e34318ea714cd0e380) and approve the pull request when available

Reply:
@f0uriest [d028bdd](https://github.com/PlasmaControl/DESC/pull/1568/commits/d028bdd6c3a15cfc414f26e34318ea714cd0e380) causes oom